studentlife/settings/production.py

```diff
@@ -1,43 +1,32 @@
 import os
 
-# import sentry_sdk
-# from sentry_sdk.integrations.django import DjangoIntegration
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
 
-from studentlife.settings.base import *  # noqa
+from studentlife.settings.base import *  # noqa: F401, F403
+from studentlife.settings.base import BACKEND_DOMAIN, DATABASES
 
 
-DEBUG = True
+DEBUG = False
```

Comment:
For now we should keep debug to true, since this isn’t a prod app yet and we’ll be changing all our credentials before it goes into production. Just in case iOS has more issues it’s easier to debug.

Reply:
Makes sense